app.js

```diff
@@ -0,0 +1,50 @@
+let books = [];
+let id = 0;
+
+const form = document.getElementById('form');
+function displayData() {
+  books = JSON.parse(localStorage.getItem('books'));
+  const bookSection = document.getElementById('books');
+  bookSection.innerHTML = ''; // Clear the existing content
+  books.forEach((book, index) => {
+    const bookElement = document.createElement('div');
+    bookElement.innerHTML = `
+      <p id="title">${book.title}</p>
+      <p id="author">${book.author}</p>
+      <button class='btn' onclick='removeBook(${index})'>Remove</button>
+      <hr />`;
+    bookSection.appendChild(bookElement);
+  });
+}
```

Comment:
- [ ] Please 🙏🏻 cross-check your work before submitting the project for review as I can see that your code is not working and have some issue which results in **console** *errors* and seems like you just used someone's code copy-pasted it and didn't check even whether it is working on not. Having the console error in the projects looks unprofessional and causes `bad-user-experience`. Kindly fix the issue and have a look at the below image.

![Screenshot from 2023-07-04 15-39-58](https://github.com/Wils-FOURR-speed/Awesome-Books/assets/66476329/f98688db-83d8-412f-9992-b8343be82ce6)